src/mod.ts

```diff
@@ -1,25 +1,28 @@
-import { BUILDER_TAG, HOOKS_TAG, RUNTIME_TAG } from "./libraries.ts";
+import { RUNTIME_TAG } from "./libraries.ts";
 import { getStartHookAdditionalDenoFlags } from "./flags.ts";
+export { getProtocolInterface } from "./deps.ts";
+export type { Protocol } from "./deps.ts";
 
 export const projectScripts = (args: string[]) => {
   const startHookFlags = getStartHookAdditionalDenoFlags(args);
   return {
     "runtime": "deno",
     "hooks": {
       "get-manifest":
-        `deno run -q --config=deno.jsonc --allow-read --allow-net --allow-env https://deno.land/x/${BUILDER_TAG}/mod.ts --manifest`,
+        `deno run -q --config=deno.jsonc --allow-read --allow-net --allow-env ${import.meta.url}/../get_manifest.ts`,
```

Comment:
What do you think about this approach to have deno-slack-hooks reference itself? Rather than hard-coding the URL to the hooks repo to match a specific tag, by using `import.meta.url` we can self-reference the URL this repo loads itself from. This also makes it easier to test this repo's hooks out from e.g. github branches.

Might need to double check the releasing steps for this repo and see if the `version.ts`-based release process is still needed or not.

Reply:
It's potentially really cool! Could we isolate this change to it's own PR? I think I'd want to do some additional testing outside the scope of this PR